src/components/StyleGuide/ColorCard.tsx

```diff
@@ -204,11 +1002,13 @@ export const DataTable = (props: React.PropsWithChildren<DataTableProps>) => {
       td: {
         fontSize: "text.tag",
         ...cellStyles,
+        borderTop: "1px solid !important",
```

Comment:
Ideally, the border should be applied to the bottom of the elements, as there should be a bottom border for the last row.  But that would mean you'll probably need to also add some styles for thead to ensure that the header row also has a bottom border.